dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/metrics/MetricsServiceBean.java

```diff
@@ -101,24 +101,60 @@ public long dataversesPastDays(int days) throws Exception {
     /**
      * Datasets
      */
-    public List<DatasetsMetrics> countPublishedDatasets() {
-        return mapToDatasetsMetrics(em.createNativeQuery(
+    public List<ChartMetrics> countPublishedDatasets() {
+        return mapToChartMetrics(em.createNativeQuery(
                 "SELECT\n" +
                         "    EXTRACT(YEAR FROM dsv.lastupdatetime) as year,\n" +
                         "    EXTRACT(MONTH FROM dsv.lastupdatetime) as month,\n" +
                         "    count (dsv.id)\n" +
                         "    FROM datasetversion dsv\n" +
                         "    WHERE\n" +
                         "        dsv.versionnumber = 1 and\n" +
-                        "        dsv.minorversionnumber = 0 and\n" +
-                        "        dsv.versionstate = 'RELEASED'\n" +
+                        "        dsv.releasetime is not null\n" +
```

Comment:
I'm not sure if it will be correct. If dataset will have two published versions then it should be counted only once. But here it probably be counted twice.

Reply:
`dsv.versionnumber = 1` filter takes care of that. We define published dataset as a dataset that has version 1 and (for sanity) we also check if releasetime is preset which it should always be.

Reply:
And what if one version is 1.0 and the second one 1.1?

Reply:
You're right.